LevEditor/src/Panels/SettingsPanel.cpp

```diff
@@ -78,11 +79,89 @@ namespace LevEngine::Editor
                 
                 ImGui::TreePop();
             }
-            
             ImGui::TreePop();
         }
 
+        ImGui::Separator();
+        
+        if (ImGui::TreeNodeEx("Physics Settings", treeNodeFlags, "Physics Settings"))
+        {
+            if (ImGui::TreeNodeEx("Layer Collision Matrix", treeNodeFlags, "Layer Collision Matrix"))
+            {
+                const char* columnNames[] = {
+                    "", "L 0", "L 1", "L 2", "L 3", "L 4",
+                        "L 5", "L 6", "L 7", "L 8", "L 9"
+                };
+                
+                constexpr int columnsCount = IM_ARRAYSIZE(columnNames);
+                constexpr int rowsCount = columnsCount - 1;
+
+                static bool tableStorage[columnsCount * rowsCount] = {};
+
+                constexpr ImGuiTableFlags tableFlags = ImGuiTableFlags_SizingFixedFit
+                                                     | ImGuiTableFlags_BordersOuter
+                                                     | ImGuiTableFlags_BordersInnerH
+                                                     | ImGuiTableFlags_Hideable
+                                                     | ImGuiTableFlags_BordersInner
+                                                     | ImGuiTableFlags_NoHostExtendX;
+
+                if (ImGui::BeginTable("table", columnsCount, tableFlags))
```

Comment:
Тут лучше осмысленный айди дать, по типу
```suggestion
                if (ImGui::BeginTable("CollisionMatrix", columnsCount, tableFlags))
```

Reply:
Исправил